app/controllers/application_controller.rb

```diff
@@ -219,6 +219,7 @@ def require_admin
   end
 
   def render_403
+    return head(:forbidden) unless respond_to?(:layout)
```

Comment:
We use this method in unattended controller where there is no layout set, which causes `Undefined method: layout` instead of 403. Will take a look, thanks.